kong/plugins/acme/api.lua

```diff
@@ -125,7 +125,7 @@ return {
     end,
 
     PATCH = function()
-      ngx_timer_at(0, client.renew_certificate)
+      ngx_timer_at(0, handler.renew)
```

Comment:
Do we need to start timer here? What if we started it here:
https://github.com/Kong/kong/blob/master/kong/plugins/acme/handler.lua#L95

Reply:
Ok, figured it out, the sole purpose of PATCH is to renew (on control plane, which feels strange).